concepticondata/conceptlists/Wu-2020-150.tsv

```diff
@@ -0,0 +1,151 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Wu-2020-150-1	1	one	1493	ONE
+Wu-2020-150-2	2	water	948	WATER
+Wu-2020-150-3	3	two	1498	TWO
+Wu-2020-150-4	4	dog	2009	DOG
+Wu-2020-150-5	5	fish	227	FISH
+Wu-2020-150-6	6	tongue	1205	TONGUE
+Wu-2020-150-7	7	eye	1248	EYE
+Wu-2020-150-8	8	ear	1247	EAR
+Wu-2020-150-9	9	fire	221	FIRE
+Wu-2020-150-10	10	blood	946	BLOOD
+Wu-2020-150-11	11	stone	857	STONE
+Wu-2020-150-12	12	see	1409	SEE
+Wu-2020-150-13	13	bone	1394	BONE
+Wu-2020-150-14	14	skin	763	SKIN
+Wu-2020-150-15	15	name	1405	NAME
+Wu-2020-150-16	16	tooth	1380	TOOTH
+Wu-2020-150-17	17	nose	1221	NOSE
+Wu-2020-150-18	18	star	1430	STAR
+Wu-2020-150-19	19	die	1494	DIE
+Wu-2020-150-20	20	come	1446	COME
+Wu-2020-150-21	21	head	1256	HEAD
+Wu-2020-150-22	22	hear	1408	HEAR
+Wu-2020-150-23	23	woman	962	WOMAN
+Wu-2020-150-24	24	path	2252	PATH
+Wu-2020-150-25	25	mouth	674	MOUTH
+Wu-2020-150-26	26	breast	1402	BREAST
+Wu-2020-150-27	27	night	1233	NIGHT
+Wu-2020-150-28	28	eat	1336	EAT
+Wu-2020-150-29	29	you	1215	THOU
+Wu-2020-150-30	30	moon	1313	MOON
+Wu-2020-150-31	31	smoke	778	SMOKE (EXHAUST)
+Wu-2020-150-32	32	hair	1040	HAIR
+Wu-2020-150-33	33	bird	937	BIRD
+Wu-2020-150-34	34	black	163	BLACK
+Wu-2020-150-35	35	fly	1441	FLY (MOVE THROUGH AIR)
```

Comment:
Could also be the fly.

Reply:
As the items of this list are claimed to be the most frequently occurring English glosses across dictionaries which overlap considerably (if not completely if we are to believe the authors) with Swadesh's lists and Leipzig-Jakarta, I'd stick to FLY (MOVE THROUGH AIR).